app/models/shopping_list_item.rb

```diff
@@ -1,38 +1,39 @@
 # frozen_string_literal: true
 
 class ShoppingListItem < ApplicationRecord
-  belongs_to :shopping_list
+  belongs_to :list, class_name: 'ShoppingList'
```

Comment:
Thank god we can stop it with the `shopping_list_item.shopping_list` and the `shopping_list.shopping_list_items` stuff now.